### DISCUSSION TITLE
自己构建的程序，前端没有中文语言，请问有空上传一下中文的json吗

### DISCUSSION THREAD

**Discussion by Jikexiaobai** [AlistGo/alist#2084] [General]
自己构建的程序，前端只有英语，请问有空上传一下中文的json吗

![image](https://user-images.githubusercontent.com/49304819/197333388-a4ebb9a4-590d-4a57-88f9-747b446b2bff.png)

**Comment by anwen-anyi** [comment]
【【Alist】Alist v3 自编译教程丨手把手教你从"0"编译全过程-哔哩哔哩】 https://b23.tv/h03MJEL

**Comment by naicfeng** [comment]
```
cd alist-web
wget https://crowdin.com/backend/download/project/alist/zh-CN.zip
unzip zh-CN.zip
node ./scripts/i18n.mjs
```

**Comment by Jikexiaobai** [comment]
谢谢
